.releaserc.js

```diff
@@ -1,3 +1,31 @@
+const { parserOpts, releaseRules } = require('./scripts/semantic-commits/change-categories')
+
 module.exports = {
-  ...require('./.releaserc.base'),
+  plugins: [
+    ['@semantic-release/commit-analyzer', {
+      preset: 'angular',
```

Comment:
preset angular? That seems odd?

Reply:
Yeah it does. The `@semantic-release/commit-analyzer` uses angular by default. Our release rules are built on top of the angular release rules and we use the angular changelog syntax. Under the hood, this tool expects a preset, a custom npm configuration or will fallback to angular: https://github.com/semantic-release/commit-analyzer/blob/master/lib/load-parser-config.js#L20